cekit-helpers/cekit-cache-server/src/test/java/org/kie/cekit/cacher/builds/github/NightlyBuildsPullRequestAcceptorTest.java

```diff
@@ -46,24 +46,24 @@ public void testNightlyRhpamCommentaryAddition() throws Exception {
         Module bcMonitoring = yamlFilesHelper.load(bcMonitoringFile);
         yamlFilesHelper.writeModule(bcMonitoring, bcMonitoringFile);
         buildUtils.reAddComment(bcMonitoringFile, "name: \"" + buildUtils.RHPAM_BUSINESS_CENTRAL_MONITORING_DISTRIBUTION_ZIP + "\"",
-                String.format("  # %s", "rhpam-7.8.0.redhat-20191006-monitoring-ee7.zip"));
-        Assertions.assertTrue(containsComment(bcMonitoringFile, String.format("  # %s", "rhpam-7.8.0.redhat-20191006-monitoring-ee7.zip")));
+                String.format("  # %s", "rhpam-7.8.0.redhat-201006-monitoring-ee7.zip"));
```

Comment:
just this date format "-201006-" that I’m not sure if it’s right in the rest everything looks great :)

Reply:
that`s correct, they build date was changed from yyyyMMdd to yyMMdd